LEAF_Request_Portal/templates/reports/LEAF_import_data.tpl

```diff
@@ -332,29 +335,29 @@
         console.log(requestData);
         var title = $('input[name="toggle"]:checked').attr('id') === 'newFormToggler' ? titleInputNew.val() : titleInputExisting.val();
 
-        if (requestData['failed'] !== undefined) {
+        if (typeof (requestData['failed']) !== "undefined") {
             failedRequests.push(requestData['failed']);
         } else {
             portalAPI.Forms.newRequest(
                 categoryID,
                 requestData,
                 function (recordID) {
 
-                    // recordID is the recordID of the newly created request, it's 0 if there was an error
+                    /* recordID is the recordID of the newly created request, it's 0 if there was an error */
                     if (recordID > 0) {
                         createdRequests++;
                         requestStatus.html(createdRequests + ' out of ' + (sheet_data.cells.length - 1) + ' requests completed, ' + failedRequests.length + ' failures.');
 
-                        //if (changeToInitiator !== undefined && changeToInitiator != null) {
-                        // set the initiator so they can see the request associated with their availability
+                        /*if (changeToInitiator !== undefined && changeToInitiator != null) { */

```

Comment:
Might want to delete this commented code

Reply:
Addressed in commit `7c5`